js/latoken.js

```diff
@@ -0,0 +1,715 @@
+            let result = {};      
+            result.id = currencies['currencyId'];
+            result.symbol = currencies['symbol'];
+            result.name = currencies['name'];
+            result.precision = currencies['precission'];
+            result.type = currencies['type'];
+            result.fee = currencies['fee'];
+            return {
+                'id': result.id,
+                'symbol': result.symbol,
+                'name': result.name,
+                'precision': result.precision,
+                'type': result.type,
+                'fee': result.fee,
+            }
+        } 
+        const currencies = await this.publicGetExchangeInfoCurrencies();
+        let result = [];      
+            for (let i = 0; i < currencies.length; i ++) {  
+                let currency = currencies[i];
+                let id = currency['currencyId'];
+                let symbol = currency['symbol'];
+                let name = currency['name'];
+                let precision = currency['precission'];
+                let type = currency['type'];
+                let fee = currency['fee'];
+                result.push({
+                    'id': id,
+                    'symbol': symbol,
+                    'name': name,
+                    'precision': precision,
+                    'type': type,
+                    'fee': fee,
+                }) 
+            }
+        return result;
+    }
+
+    parseOneTrade (trade) {
+        let side = this.safeString (trade, 'side');
+        let price = parseFloat (trade['price']);
+        let amount = parseFloat (trade['amount']);
+        let timestamp = this.safeValue(trade,'timestamp');
+        return { 
+            'side': side,
+            'price': price,
+            'amount': amount,
+            'timestamp': timestamp, 
+        }
+    }
+
+    parseTrades (trades, sideKey = 0, priceKey = 0, amountKey = 0, timestampKey = 0) {
+        return Object.values (trades || []).map (trade => this.parseOneTrade (trade, sideKey, priceKey, amountKey, timestampKey))
+    }
+
+    parseTrade (trades, tradeKey = 'trades', sideKey = 0, priceKey = 0, amountKey = 0, timestampKey = 0) {
+        return {
+            'pairId': trades['pairId'],
+            'symbol': trades['symbol'],
+            'tradeCount': trades['tradeCount'],
+            'trades': (tradeKey in trades) ? this.parseTrades (trades[tradeKey], sideKey, priceKey, amountKey, timestampKey) : null,
+        }
+    }
+
+    async fetchTrades (symbol, limit = 100, params = {}) {
+        await this.loadMarkets();
+        let market = this.market (symbol);
+        let resp = {
+            'symbol': market['symbol'],
+            'limit': limit,
+        };
+        let response = await this.publicGetMarketDataTrades (this.extend (resp, params));
+        let trades = this.parseTrade(response) 
+        return trades;
+    }
+
+    parseOneOrderTrade (trade) {
```

Comment:
We should use `parseTrade` with the `parseTrades` base method for all trades, as, for example, done here: https://github.com/ccxt/ccxt/blob/master/js/kucoin.js#L888